project1/NOTES.md

```diff
@@ -0,0 +1,25 @@
+TASK-1: Postgresql
```

Comment:
Time elapsed for task-1 is missing. Probably in this task you may not have any questions or tricky steps or online references to include. But from the next task it is important to include all the steps i.e process of you have done, online references, questions you addressed during the task and tricky/consuming steps

Reply:
i have updated the task